### ISSUE TITLE
Smoke tests freetypeOnlyBundledOnWindowsAndMacOS failed with alpine jdk 16 

### ISSUE THREAD

**Issue by sophia-guo** [adoptium/temurin-build#2579]
Smoke tests freetypeOnlyBundledOnWindowsAndMacOS failed with alpine jdk 16 
https://ci.adoptopenjdk.net/job/build-scripts/job/jobs/job/jdk16u/job/jdk16u-alpine-linux-x64-hotspot_SmokeTests/4/console

```
20:26:12  FAILED: freetypeOnlyBundledOnWindowsAndMacOS
20:26:12  java.lang.AssertionError: Expected libfreetype not to be bundled but it is. expected [0] but found [1]
20:26:12  	at org.testng.Assert.fail(Assert.java:96)

```

Is this expected @adamfarley

**Comment by sophia-guo** [comment]
Same issue to jdk16 aix openj9:
https://ci.adoptopenjdk.net/job/build-scripts/job/jobs/job/jdk16u/job/jdk16u-aix-ppc64-openj9_SmokeTests/4/console

jdk11 aix openj9 passed.
https://ci.adoptopenjdk.net/job/build-scripts/job/jobs/job/jdk11u/job/jdk11u-aix-ppc64-openj9_SmokeTests/4/console

**Comment by adamfarley** [comment]
@sophia-guo - We only expect freetype to be bundled on Windows and Mac, to my knowledge. I hadn't heard about alpine or aix bundling it.

**Comment by adamfarley** [comment]
Not sure about alpine, but the aix one looks like a genuine bug. Will investigate why we're bundle freetype there.

I mean, we don't bundle it on jdk11u for aix.

**Comment by adamfarley** [comment]
Odd. We specify --skip-freetype on both jdk11u and jdk16u, however...

On jdk11u we see this:
```
19:18:04  checking for FREETYPE... yes
19:18:04  checking for freetype... yes (using pkg-config)
19:18:04  Using freetype: system
```

And on jdk16u we see this:
```
00:36:16  Using freetype: bundled
```

Which is...odd. 

Did the default behaviour change to "bundled" on JDK16? No, because then it's be happening on linux too, and it [doesn't](https://ci.adoptopenjdk.net/job/build-scripts/job/jobs/job/jdk16u/job/jdk16u-linux-x64-hotspot/19/consoleFull).

Could it be that the system freetype is missing on one of our aix machines, and we default to building our own "bundled" freetype when that happens? Unknown, will check build history and consult those with machine access.

**Comment by adamfarley** [comment]
Conclusions for the 2 platforms we're seeing this issue on:

**AIX (JDK16) **

For JDK13 and upwards, OpenJDK defaults to using a bundled freetype on aix builds (in addition to Windows and Mac).

Evidence: https://bugs.openjdk.java.net/browse/JDK-8218046

Unless we plan to override this, we'll need to update the test. I'll probably change the name too.

**Alpine**

Adopt requires a bundled freetype in the config args for Alpine builds.

This is because we don't specify the --skip-freetype option when calling makejdk-any-platform.sh, which (in turn) is because we don't include --skip-freetype in [alpine-linux.sh](https://github.com/AdoptOpenJDK/openjdk-build/blob/master/build-farm/platform-specific-configurations/alpine-linux.sh).

So it could be a deliberate omission, or it could be accidental.

Based on [this comment}(https://github.com/AdoptOpenJDK/openjdk-docker/issues/529#issuecomment-811403244), I think it was an accident, as it seems to state that we shouldn't be bundling freetype on alpine.

**Summary** 

I'm going to create seperate PRs for these issues, given the disparate nature of the fixes. Will update with links after my next meeting

**Comment by adamfarley** [comment]
AIX PR: https://github.com/AdoptOpenJDK/openjdk-build/pull/2585
Alpine PR: https://github.com/AdoptOpenJDK/openjdk-build/pull/2587

Also, in the process of handling this defect, I noticed that it's theoretically impossible to set the freetype lib directory manually on Windows. It's a simple fix, so I created a [PR for that too](Improvement PR: https://github.com/AdoptOpenJDK/openjdk-build/pull/2588). This isn't a priority though, and can wait till after the release. This issue can be closed before it's complete too, just mentioning it here to link things together.
